Appraisals

```diff
@@ -0,0 +1,18 @@
+appraise "rails-4.0" do
+  gem "rails", "4.0.2"
```

Comment:
`~> 4.0` would mean `>= 4.0, < 5`, which I don't think is what you want. I can do `~> 4.0.2` or `-> 4.0.0` if you find that cleaner